content/src/content/jcr_root/apps/core/wcm/components/documentcloudviewer/v1/documentcloudviewer/clientlibs/editor/.content.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[cq.authoring.dialog]"/>
```

Comment:
does it needs to be included in all dialogs? may be make it something like `core.wcm.components.documentcloudviewer.v1.editor` and add it to dialog through `extraClientLibs`. The same way it is done for other components.

Reply:
Done in 456c69ef12904f80688805839c1386fed14fee04